### PR TITLE
Improve UX of non-interactive slider

### PR DESCRIPTION
**Pull request by Pythongor**
## Description

This pull request makes the slider look more logical and compliant with UX standards. Now in the inactive state the slider looks more subdued and the cursor on the hover is displayed correctly.

Closes: #7813

### CODE REVIEW

**Comment by shubhamofbce** [comment]
@Pythongor  Can you please add a comparison screenshot after the fix here, so we can take a look without running the code.

**Comment by pngwn** [comment]
Looks great!

Screenshot 2024-03-25 at 14 34 26

**Comment by pngwn** [comment]
Fixed the stories and also fixed a small bug that mean the background size wasn't being coloured in when the max was not set.

File: js/slider/Index.svelte [hunk 1]
```diff
@@ -61,11 +61,10 @@
...
 	function setSliderRange(): void {
-		rangeInput.style.backgroundSize =
-			((Number(rangeInput.value) - Number(rangeInput.min)) /
-				(Number(rangeInput.max) - Number(rangeInput.min))) *
-				100 +
-			"% 100%";
+		const dividend = Number(rangeInput.value) - Number(rangeInput.min);
+		const divisor = Number(rangeInput.max) - Number(rangeInput.min);
+		const h = divisor === 0 ? 0 : dividend / divisor;
+		rangeInput.style.backgroundSize = h * 100 + "% 100%";
```
Comment (pngwn) [inline js/slider/Index.svelte:67]: Just added some checks so we don't get infinity.

File: js/slider/Index.svelte [hunk 2]
```diff
@@ -24,7 +24,7 @@
 	export let scale: number | null = null;
 	export let min_width: number | undefined = undefined;
 	export let minimum: number;
-	export let maximum: number;
+	export let maximum = 100;
```
Comment (pngwn) [inline js/slider/Index.svelte:27]: Default for browser is 100 anyway, so just set this explicitly, to avoid undefined values.

**Review by pngwn — APPROVED** [review]
Looks great. Thanks for tackling this @Pythongor!

**Pull request merged**